src/app/shared/services/http.service.ts

```diff
@@ -202,13 +203,9 @@ export class HttpService {
     return this.http.get<OptionsSettings>('api/testtool/reset').pipe(catchError(this.handleError()));
   }
 
-  reset(): Observable<void> {
-    return this.http.post<void>('api/runner/reset', {}).pipe(catchError(this.handleError()));
-  }
-
-  runReport(storage: string, targetStorage: string, reportId: string): Observable<void> {
+  runReport(storage: string, reportId: string): Observable<TestResult> {
     return this.http
-      .post<void>('api/runner/run/' + storage + '/' + targetStorage + '/' + reportId, {
+      .post<void>('api/runner/run/' + storage + '/' + reportId, {
```

Comment:
Can you make this a string template:
```suggestion
      .post<void>(`api/runner/run/${storage}/${reportId}`, {
```